apps/backend/src/libs/plugins/libs/helpers/token-user-extractor.ts

```diff
@@ -0,0 +1,28 @@
+import { type JWTPayload } from "jose";
+
+import { type UserAuthResponseDto } from "../../../../modules/users/libs/types/types.js";
```

Comment:
Use absolute imports to take another module
```suggestion
import { type UserAuthResponseDto } from "~/modules/users/libs/types/types.js";
```